src/domain/openvino_models.py

```diff
@@ -0,0 +1,64 @@
+"""This module implements OpenVINO related classes and functionalities."""
+
+import numpy as np
+import openvino as ov
+import openvino.properties as props
+from PytorchWildlife.models import detection as pw_detection
+import torch
+
+core = ov.Core()
+core.set_property({props.cache_dir: "cache"})
+# TODO: Add NPU specific properties
+
+
```

Comment:
Add logger like other classes, so that we can use it and embedd it in UI (or daemon later). Plus, consider logging debug or info messages to have feedback on progress